spec/requests/partner_users_requests_spec.rb

```diff
@@ -31,6 +42,8 @@
 
     context "with valid user params" do
       it "invites a new user and redirects back with notice" do
+        user.add_role(:org_admin)
```

Comment:
Can you move this to a `before(:each)` block?

Reply:
Fixed it here [129a2b8](https://github.com/rubyforgood/human-essentials/pull/4468/commits/129a2b83d41257311ae379f3b2ebed7a75537659)